### ISSUE TITLE
[Error 39] Directory not empty (again)

### ISSUE THREAD

**Issue by andreanuzzo** [picrust/q2-picrust2#7]
Hi Gavin, 

I am having problems with the q2-picrust2 plugin, version 0.0.3 on a new qiime2-2019.4 environment. After the install (which, by the way removes a lot of necessary plugins, such as q2-DADA2), I get again a similar error as https://github.com/picrust/picrust2/issues/44 when trying to run the following (with the test tables):

`qiime picrust2 full-pipeline --i-table mammal_biom.qza \
                             --i-seq mammal_seqs.qza \
                             --output-dir q2-picrust2_output \
                             --p-threads 20 \
                             --p-hsp-method pic \
                             --p-max-nsti 2`

Which throws the error:

`Plugin error from picrust2:

  [Errno 39] Directory not empty: blabla/users/andy/tmp/tmpd89n2chr'

Debug info has been saved to blabla/users/andy/tmp/qiime2-q2cli-err-3fvp9dj4.log`

I solved it again by adding a `pass` at line 482 of `shutil.py` (in anaconda3/envs/qiime2-2019.4/lib/python3.6/shutil.py) as I did previously. It worked and produced the output, but I would like to transfer the fix you applied couple of months ago in the full picrust2 pipeline, can you help?

Best,

**Comment by gavinmdouglas** [comment]
Thanks for reporting this @andreanuzzo , this must be a similar problem as before, but not identical - the original problem should now be resolved in the q2-picrust2 0.0.3.

To help troubleshoot this issue would you mind running the standalone PICRUSt2 with the ```--verbose``` option and reporting the output? To do that you would need to export the QZA files (with ```qiime tools export```) and run:

```
picrust2_pipeline.py -s FASTA -i BIOM -o picrust2_out_pipeline -p 20 --verbose
```

**Comment by gavinmdouglas** [comment]
Closed due to inactivity.
